catalyst_voices/packages/catalyst_voices_shared/lib/src/dynamic_padding/dynamic_padding.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:catalyst_voices_shared/src/responsive_builder/responsive_breakpoint_key.dart';
+import 'package:catalyst_voices_shared/src/responsive_builder/responsive_builder.dart';
+import 'package:flutter/widgets.dart';
+
+// A [DynamicPadding] is a StatelessWidget that applies a padding based on the
+// current screen size.
+//
+// The widget wraps its [child] in a ResponsiveBuilder that calculates the 
+// proper padding value based on the screen size and wraps it again in a Padding
+// to display the selected padding value.
+//
+// The possible arguments are [xs], [sm], [md], [lg], [other] following the 
+// Material design standards and the ResponsiveBuilder arguments.
+// Each screen size has a default value to simplify widget usage.
+//
+// Example usage:
+// 
+// ```dart
+// DynamicPadding(
+//   xs: const EdgeInsets.all(4.0),
+//   sm: const EdgeInsets.all(6.0),
+//   md: const EdgeInsets.only(top: 6.0),
+//   lg: const EdgeInsets.symmetric(vertical: 15.0),
+//   child: Text('This is an example text with dynamic padding.')
+// );
+// ```
+
+class DynamicPadding extends StatelessWidget {
```

Comment:
We already have `ResponsiveBuilder` and  `ResponsiveBreakpointKey` so I'd like to propose renaming `DynamicPadding` to `ResponsivePadding` to keep things consistent and explicit. 
What do you think?